v4-client-js/examples/composite_example.ts

```diff
@@ -37,7 +37,7 @@ async function test(): Promise<void> {
         side,
         price,
         0.01,
-        randomInt(100_000_000),
+        randomInt(MAX_CLIENT_ID),
```

Comment:
Suggestion: can we abstract this into a function? Maybe call it `getRandomClientId`?